codegen/module_system.go

```diff
@@ -1355,84 +1415,126 @@ func (generator *GatewayServiceGenerator) ComputeSpec(
 func (generator *GatewayServiceGenerator) Generate(
 	instance *ModuleInstance,
 ) (*BuildResult, error) {
-	service, err := generator.templates.ExecTemplate(
-		"service.tmpl",
-		instance,
-		generator.packageHelper,
-	)
-	if err != nil {
-		return nil, errors.Wrapf(
-			err,
-			"Error generating service service.go for %s",
-			instance.InstanceName,
+	var fileMap sync.Map
+	var wg sync.WaitGroup
+	wgCount := 5
```

Comment:
Why is this hard coded to 5?

Reply:
added a comment to specify its a goroutine count